spec/integrations/user_index_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'rails_helper'
+
+RSpec.describe 'User #Index Page', type: :feature do
+  before(:each) do
+    @user1 = User.create(name: 'Salomon Tshobowa', photo: 'https://salomon.jpg',
+                         bio: 'Micronaute', posts_counter: 5)
+    @user2 = User.create(name: 'Chere Lemma', photo: 'https://chere.jpg',
+                         bio: 'Micraunote', posts_counter: 5)
+    visit users_path
+  end
+
```

Comment:
- [ ] According to project requirements it should be Tests that assert you can see the **username** of all other users in **user index page**. Please fix to make your test work as expected 😄